### ISSUE TITLE
signInWithCredential on Android/iOS has different params

### ISSUE THREAD

**Issue by Victoor** [davecoates/rn-firebase-bridge#13]
In the readme file it says to pass the credential like these:

```
const credential = FacebookAuthProvider.credential(token);
signInWithCredential(credential);
```

But in Android `FirebaseBridgeAuth.java` the function it's defined as `void signInWithCredential(String id, final Promise promise)`.

Maybe I'm wrong but I'm having problems making this works on Android. 😕

**Comment by davecoates** [comment]
Sorry about that - what  I wrote in the docs wasn't quite correct. Everything over the bridge is async so `FacebookAuthProvider.credential(token)` returns a promise and `signInWithCredential` expected you to await the promise value and pass that through, eg.

`FacebookAuthProvider.credential(token).then(signInWithCredential)`

I've updated it so you  can do either now - if you pass through the promise it will now work.

Let me know if that works for you now
